packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -10,9 +10,9 @@
   },
   "main": "out/src",
   "dependencies": {
-    "@salesforce/core": "5.3.20",
+    "@salesforce/core": "6.4.3",
```

Comment:
@daphne-sfdc please bump this to the most recent version of v6. I know its a moving target, but lets try to get this merged being as close to current as possible.

Reply:
Probably should visit all of the one to see if they have been bumped